pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -970,6 +973,7 @@
     "zodiac" = ps: with ps; [ ];
     "zone" = ps: with ps; [ ];
     "zoneminder" = ps: with ps; [ zm-py ];
-    "zwave" = ps: with ps; [ homeassistant-pyozw pydispatcher ];
+    "zwave" = ps: with ps; [ aiohttp-cors homeassistant-pyozw paho-mqtt pydispatcher ]; # missing inputs: python-openzwave-mqtt[mqtt-client]
```

Comment:
Unless someone packages `python-openzwave-mqtt==1.4.0` (https://github.com/cgarwood/python-openzwave-mqtt) we are losing zwave support.

Reply:
This might affect a few people. Not me so.

Reply:
Packaging this, as it is also required for testing `config`.

Reply:
I'll give it a shot.

Reply:
Already done and pushed.

Reply:
From the release notes:

> Z-Wave
>
> The old zwave integration is now considered legacy and deprecated. This integration is still based on OpenZwave 1.4, which has been out of active maintenance for quite some time now.
>
> Alternatively, you can move to one of the other Z-Wave solutions Home Assistant offers:
>
>   -  OpenZWave (currently not maintained)
>
>   -  Z-Wave JS (introduced in this release, recommended)